backtester/data/fake.go

```diff
@@ -0,0 +1,67 @@
+package data
```

Comment:
I think it might be better adding all the struct and methods to a test file.

Reply:
roger that, updated